src/benchmark/CMakeLists.txt

```diff
@@ -3,14 +3,11 @@ add_executable(
 
     benchmark_basic_fixture.cpp
     benchmark_basic_fixture.hpp
-    benchmark_join_fixture.cpp
-    benchmark_join_fixture.hpp
     benchmark_main.cpp
-    benchmark_template.cpp
+    # benchmark_template.cpp
```

Comment:
I removed it when I removed all benchmarks and re-added them one by one and then forgot about that! But on that note, why do we have this in the here anyway (in the CMakeLists)? Because there is nothing to compile..

Reply:
No idea. I'm not even sure if we need that template. I guess most people do it like me and try to find the smallest running benchmark to use that as a starting point.